src/HonzaBotner.Discord.Services/Jobs/TriggerRemindersJobProvider.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using Hangfire;
+using HonzaBotner.Database;
+using HonzaBotner.Discord.Extensions;
+using HonzaBotner.Discord.Services.Options;
+using HonzaBotner.Services.Contract;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace HonzaBotner.Discord.Services.Jobs
+{
+    public class TriggerRemindersJobProvider : IRecurringJobProvider
+    {
+        private readonly IServiceScopeFactory _factory;
+
+        private readonly ILogger<TriggerRemindersJobProvider> _logger;
+
+        private readonly ReminderOptions _options;
+
+        private readonly DiscordWrapper _discord;
+
+        private readonly IGuildProvider _guild;
+
+        public TriggerRemindersJobProvider(
+            IServiceScopeFactory factory,
+            ILogger<TriggerRemindersJobProvider> logger,
+            IOptions<ReminderOptions> options,
+            DiscordWrapper discord,
+            IGuildProvider guild
+        )
+        {
+            _factory = factory;
+            _logger = logger;
+            _options = options.Value;
+            _discord = discord;
+            _guild = guild;
+        }
+
+        public string GetKey() => "reminders-trigger";
```

Comment:
```suggestion
        public string GetKey => "reminders-trigger";
```
Also it needs appropriate changes to code